testsuite/tests/lib-unix/unix-socket/is-linux.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# This script is related to the 'recvfrom_linux.ml' test.
+
+# Workaround: the next tests (bytecode and native) depend on
+# stdout redirection, but running a script sets both of
+# those to the empty string. Clear them here:
```

Comment:
I don't really follow what's going on here. Could you clarify?

Reply:
This one's weird, I think it's a bug in ocamltest (but maybe it's intentional):

https://caml.inria.fr/mantis/view.php?id=7910

Now that I've filed the issue I can put a link in this comment for future travelers.

Reply:
(cc @shindere)

Reply:
Thanks for opening a ticket and adding a link